.buildkite/beta-builds.yml

```
@@ -20,7 +14,7 @@ steps:
   - label: "Gradle Wrapper Validation"
     command: |
       validate_gradle_wrapper
-    plugins: *common_plugins
+    plugins: [$CI_TOOLKIT]
```

Nice tidy up.

Have you considered:

```suggestion
    plugins: $CI_TOOLKIT
```

I never tried it but I assume the Buildkite parser would be smart enough to allow a scalar value here and convert it to a sequence internally.

However, maybe we're better off leaving the `[ ]` so the diff when adding a new plugin will be smaller 🤷‍♂️

---

While I think that Buildkite would allow a single value in an attribute expecting an array (and would wrap it in an array automatically for us) and thus allow such a syntax… personally I prefer to be explicit in it being an array and thus keep the `[…]` syntax.